shell/initialize/entry-helpers.js

```diff
@@ -294,7 +294,7 @@ export async function mountApp(appPartials, VueClass) {
     // Push the path and let route to be resolved
     router.push(path, undefined, (err) => {
       if (err) {
-        const errorHandler = vueApp.config.errorHandler || console.error; // eslint-disable-line no-console
+        const errorHandler = vueApp?.config?.errorHandler || console.error; // eslint-disable-line no-console
```

Comment:
I've noticed a few times that this is being invoked without vueApp.config being defined.